requirements/constraints.txt

```diff
@@ -19,3 +19,4 @@ diff-cover<4.2.0
 
 # requests has a constraint idna<3 in it, so pinning it unless requests updates the constraint
 idna<3.0
+charset-normalizer~=2.0.0
```

Comment:
Please also add the note about charset-normalizer pin here that you have added in description.

Reply:
Updated the details.